src/main/kotlin/racingCar/domain/race/RaceResult.kt

```diff
@@ -0,0 +1,25 @@
+package racingCar.domain.race
+
+import racingCar.domain.car.CarStatus
+
+data class RaceResult(val roundResults: List<RaceRoundResult>) {
+    val roundCount: Int = roundResults.size
+    val winners: List<CarStatus>
+
+    init {
+        val tempWinners: MutableList<CarStatus> = mutableListOf()
+        var curMaxPosition = 0
+        for (elem in roundResults.last().carStatuses) {
+            if (curMaxPosition < elem.value.position) {
+                tempWinners.clear()
+                tempWinners.add(elem.value)
+                curMaxPosition = elem.value.position
+            } else if (curMaxPosition == elem.value.position) {
+                tempWinners.add(elem.value)
+            }
+        }
+        winners = tempWinners.toList()
+    }
+
+
+}
```

Comment:
init 구문에 작성된 코드는 다음과 같이 함수로 뺄 수 있을 것 같아요!
```suggestion
data class RaceResult(val roundResults: List<RaceRoundResult>) {
    val roundCount: Int = roundResults.size
    val winners: List<CarStatus> = findWinner(roundResults.last())
    
    private fun findWinners(result: RaceRoundResult): List<CarStatus> = with(result.carStatuses) {
        
        val winningDistance = maxOfOrNull { it.value.position } ?: 0
        
        filter { it.value.position == winningDistance }
            .map { it.value }
    }
}
```